src/Config/ValueProvider/TimeoutProvider.php

```diff
@@ -1,84 +0,0 @@
-<?php
```

Comment:
This class is no longer necessary: we pass the default timeout value (10s) in `ConfigurationFactory` and this setting was never required in `schema.json`.